.travis.yml

```diff
@@ -0,0 +1,25 @@
+sudo: false
+language: node_js
+node_js:
+  - 4
+  - 5
+  - stable
```

Comment:
That is currently stable. We could add it, but travis is already slow enough for us to give the luxury of running on the same version twice